src/tests/Interop/ICustomMarshaler/Primitives/ICustomMarshalerNative.cpp

```diff
@@ -7,27 +7,27 @@
 using TestDelegate = LPCSTR(STDMETHODCALLTYPE*)(int);
 using TestDelegateRef = LPCSTR(STDMETHODCALLTYPE*)(int*);
 
-extern "C" int DLL_EXPORT STDMETHODCALLTYPE NativeParseInt(LPCSTR str)
```

Comment:
Please keep the `STDMETHODCALLTYPE`s on the native methods without adding a `CallingConvention=` setter in managed.

Reply:
Yeah. I reread again and understand that you want like that. Done.